src/entities/course/ui/course-card/course-card.module.scss

```diff
@@ -0,0 +1,112 @@
+.course-card {
+  $accent-color: var(--accent-bg-color);
+
+  @extend %transition-all;
+
+  // cursor: pointer;
+
+  display: flex;
+  flex-direction: column;
+  // justify-content: flex-start;
+
+  width: 380px;
+  min-width: 100px;
+  height: 100%;
+  // padding: 0;
+
+  // color: $color-gray-600;
+
+  border-radius: 12px;
+
+  .card-header {
+    position: relative;
+
+    overflow: hidden;
+    display: flex;
+    gap: 24px;
+    // place-content: flex-start flex-start;
+    align-items: center;
+
+    // width: 100%;
+    height: 100%;
+    min-height: 112px;
+    padding: 0 24px;
+
+    background-color: $color-purple-50;
+    border: solid $color-gray-200;
+    border-width: 1px 1px 0 1px;
+    border-radius: 12px 12px 0 0;
+
+    &::after {
+      @extend %transition-all;
+
+      @include card-corner-accent($accent-color);
+    }
+
+    h3 {
+      margin: 0;
+    }
+
+    img {
+      // display: block;
+      width: 60px;
+      height: 54px;
+    }
+  }
+
+  .card-title {
+    width: 100%;
+    // height: auto;
+
+    font-size: 24px;
+    font-weight: $font-weight-medium;
+    // font-style: normal;
+    line-height: 32px;
+    color: $color-gray-600;
+    // text-align: left;
+    // letter-spacing: 0;
+
+    @include media-laptop {
+      font-size: 20px;
+    }
+
+    @include media-tablet {
+      font-size: 24px;
+    }
+  }
+
+  .course-info {
+    display: flex;
+    // gap: 24px;
+    align-items: center;
+    justify-content: space-between;
+
+    // width: 100%;
+    padding: 24px;
+
+    background-color: #fefefe;
```

Comment:
make a const? i see we use it in event card also

Reply:
I removed it from event card and used $color-white instead

Reply:
Then maybe we should use white here also for consistency between cards

Reply:
I don't like white color, I've found another way